lua/core/utils/status.lua

```diff
@@ -737,9 +758,18 @@ end
 -- @usage local heirline_component = astronvim.status.component.tabline_file_info()
 function astronvim.status.component.tabline_file_info(opts)
   return astronvim.status.component.file_info(astronvim.default_tbl(opts, {
-    unique_path = { hl = function(self) return { fg = self.tab_type .. "_path_fg" } end },
+    file_icon = {
+      hl = function(self)
+        local color_icons = astronvim.status.env.attributes.color_icons -- TODO: REWORK THIS
+        if type(color_icons) == "function" then color_icons = color_icons(self) end
+        if color_icons then return astronvim.status.hl.filetype_color(self) end
+      end,
+    },
+    unique_path = {
+      hl = function(self) return astronvim.status.hl.get_attributes(self.tab_type .. "_path") end,
+    },
     close_button = {
-      hl = function(self) return { fg = self.tab_type .. "_close_fg" } end,
+      hl = function(self) return astronvim.status.hl.get_attributes(self.tab_type .. "close") end,
```

Comment:
This should be `_close`. I get the following error when starting:
```
E5108: Error executing lua ...ack/packer/opt/heirline.nvim/lua/heirline/highlights.lua:33: 'buffer_activeclose_fg' is not a valid color
```

Reply:
ah thanks for catching this. Just revised the commit